src/datasources/order.datasource.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {inject} from '@loopback/core';
+import {juggler, AnyObject} from '@loopback/repository';
+const config = require('./mongo.datasource.json');
+
+export class OrderDataSource extends juggler.DataSource {
```

Comment:
Is there any particular reason for having different datasource classes for Customer and Order models? Typically, a single datasource should be shared by all models persisted by the same database.

Reply:
Yes, the reason was to use the same connection information while setting a different database name in the Class for each model. 

That said, we default the database name based on the model so I've updated the PR to just have a single DataSource shared by all models.

Reply:
> Yes, the reason was to use the same connection information while setting a different database name in the Class for each model.

I think you may be confusing database and table name?

In SQL world, a single database server can provide multiple different databases as a kind of a multi-tenancy. Each database can have multiple tables. In LoopBack, we map models to tables.

From my experience, a single application typically stores data in a single database. This make it easy to leverage SQL features like the JOIN statement. I am not sure is there are any SQL databases that would support JOIN statements across different databases.